src/modelconfig.cpp

```diff
@@ -406,6 +414,26 @@ Status ModelConfig::parseNode(const rapidjson::Value& v) {
         modelVersionPolicy = ModelVersionPolicy::getDefaultVersionPolicy();
     }
 
+    SPDLOG_DEBUG("Specified model parameters:");
+    SPDLOG_DEBUG("model_basepath: {}", getBasePath());
+    SPDLOG_DEBUG("model_name: {}", getName());
+    SPDLOG_DEBUG("batch_size: {}", getBatchSize());
+    if (isShapeAnonymous()) {
+        SPDLOG_DEBUG("shape: {}", std::string(getShapes().begin()->second));
+    } else {
+        SPDLOG_DEBUG("shape:");
+        for (auto& shapeMap : getShapes()) {
+            SPDLOG_DEBUG("  {}: {}", shapeMap.first, std::string(shapeMap.second));
+        }
+    }
+    SPDLOG_DEBUG("model_version_policy: {}", std::string(*getModelVersionPolicy()));
```

Comment:
What if `getModelVersionPolicy` returns nullptr?

Reply:
Is it possible to get nullptr if default constructor of ModelConfig creates shared pointer?

Reply:
After checking entire solution I assume it is not possible to create ModelConfig without model version policy. To protect ourselves from introducing such bug (allow creating modelconfig with no version policy ptr) later on, it's good to insert check for nullptr anyway. By doing this we dont need to remember to fix this part in the future.